tests/virt_autotest/reboot_and_wait_up_upgrade.pm

```diff
@@ -53,7 +53,10 @@ sub post_fail_hook {
     my ($self) = shift;
     reset_consoles;
     select_console('root-console');
-    $self->upload_y2logs if (check_var('offline_upgrade', 'yes'));
+    if (check_var('offline_upgrade', 'yes')) {
+        upload_logs("/root/autoupg.xml", failok => 1);
```

Comment:
I heard from Leon that /root/autoupg.xml does not exist during upgrade. I might not know the detail, @guoxuguang  may give some input here.

Reply:
It is already uploaded in the earlier step, after the /root/autoupg.xml is generated.

Reply:
I am fine if this has been uploaded.